### ISSUE TITLE
Compatibility with latest Genie and Stipple

### ISSUE THREAD

**Issue by PGimenez** [GenieFramework/StippleLatex.jl#10]
StippleLatex cannot be installed right now due to compatibility requirements with Genie and Stipple. In the [genieversion branch](https://github.com/GenieFramework/StippleLatex.jl/tree/genieversion) I bumped the versions to the current releases. It installs now but it's not usable as some things have changed in Stipple:

```julia
julia> using StippleLatex

julia> latex()
ERROR: UndefVarError: json not defined
Stacktrace:
 [1] getproperty
   @ ./Base.jl:31 [inlined]
 [2] latex(::String; expression::String, auto::Bool, display::Bool, throw_on_error::Bool, error_color::String, max_size::String, max_expand::Int64, allowed_protocols::Vector{String}, strict::String, kwargs::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
   @ StippleLatex ~/.julia/packages/StippleLatex/5JfgX/src/StippleLatex.jl:83
 [3] latex (repeats 2 times)
   @ ~/.julia/packages/StippleLatex/5JfgX/src/StippleLatex.jl:63 [inlined]
 [4] top-level scope
   @ REPL[2]:1
```

**Comment by hhaensel** [comment]
Yes I checked it already, I will follow up on StippleLatex.

**Comment by hhaensel** [comment]
I propose a new syntax.
The `latex()` will provide a span rather than a div and the `@latex` macro is replaced by a string macro.
The string macro `latex"\sin^2x"mode` provides keyword-value pairs to be used in element defintions. It accepts raw strings for the latex expression and a mode modifier.
If the expression starts with `:`, the expression is interpreted as a javascript expression, in order to link the entry to a field.

```julia
cell(class = "q-pa-md", latex":text"display)
cell(class = "q-pa-md", latex"\cos^2x"auto)
latex(class = "q-pa-md", raw"\tan^2x", display)
```

**Comment by hhaensel** [comment]
I shall also add a `@latex` macro that supports options and also provides options.

**Comment by hhaensel** [comment]
Just added `@latex()` as promised, more details [here](https://github.com/GenieFramework/StippleLatex.jl/pull/11#issuecomment-1435660528)

**Comment by hhaensel** [comment]
Should be fine from my end now.
@essenciary , @AbhimanyuAryan 
If also fine for you, go ahead and tag a new release.

**Comment by AbhimanyuAryan** [comment]
I haven't used much latex besides building resumes with overleaf latex but this looks very intuitive. Should merge @hhaensel

**Comment by PGimenez** [comment]
fixed in 5bd0ddbe7728ba62d4396354427af11bcfdd03a6
